CMakeLists.txt

```diff
@@ -7,8 +7,10 @@ enable_testing()
 macro(set_platform system_name)
     if(${system_name} MATCHES "Darwin")
         set(OS_PLATFORM "OS_DARWIN")
+        add_definitions(-DOS_DARWIN)
```

Comment:
`struct duration` can't be declared in pelikan without these definitions, since the definition of them in ccommon relies on these definitions being there. Without this, `struct duration` in cc_timer.h is just a forward declaration.